client/lobby/CBonusSelection.cpp

```diff
@@ -386,6 +388,12 @@ void CBonusSelection::goBack()
 
 void CBonusSelection::startMap()
 {
+	if(getCampaign()->formatVersion() == CampaignVersion::Chr)
+	{
+		auto * chFilesystem = new CFilesystemLoader("DATA/", VCMIDirs::get().userDataPath() / ("Hc" + std::to_string(getCampaign()->getCampId() - 20) + "Data"), 0);
+		CResourceHandler::addFilesystem("data", "Hc" + std::to_string(getCampaign()->getCampId() - 20) + "Data", chFilesystem);
+	}
+
```

Comment:
Looks like mod functionality to be honest...
Mod that provides map/campaign and is automatically activated when player selects map/campaign from a mod

Reply:
Kind of similarly to how localization mod activates only if localization matches

Reply:
Yes, that's one of the points which make this unclear. One problem is also the resource priority. If I make a chronicles mod it should also be possible to replace resources of this by mod (for example for voices in other languages).

Reply:
But basically I think the clearest way would be a import which creates a mod of the imported files.